pkg/envoy/server.go

```diff
@@ -102,6 +102,7 @@ func getXDSPath(stateDir string) string {
 func StartXDSServer(stateDir string) *XDSServer {
 	xdsPath := getXDSPath(stateDir)
 	accessLogPath := getAccessLogPath(stateDir)
+	denied403body := viper.GetString("403-msg")
```

Comment:
We configure an empty string default. Should we have a check here or just have the default say `Denied`?

Reply:
Changed the help text to "Message returned in proxy L7 403 body", but also marked this hidden as Ian suggested. Empty string will be defaulted to "Access denied" in our Envoy L7 filter.